jujugui/static/gui/src/app/utils/environment-change-set.js

```diff
@@ -178,17 +178,19 @@ YUI.add('environment-change-set', function(Y) {
           //  under. In most cases this will be `env`.
           result = callback.apply(this, arguments);
         }
-        self.fire('taskComplete', {
-          id: record.id,
-          record: record,
-          result: arguments
-        });
+        document.dispatchEvent(new CustomEvent('ecs.taskComplete', {
+          detail: {
```

Comment:
I'm assuming detail is only here so that we don't have to update everywhere in the app that expects the old YUI signature? Is it necessary or should we have a follow-up to update these?